From 51150b60c0da2ddc89e9a8690c80bf773d40f9ac Mon Sep 17 00:00:00 2001
From: HansHenrikLund <hhl@dbc.dk>
Date: Fri, 4 Jan 2013 16:06:14 +0100
Subject: [PATCH 1/2] Fjernet fejl linjer

---
 CONTENT-list.lst | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CONTENT-list.lst b/CONTENT-list.lst
index b1413fd..1f4ba9c 100644
--- a/CONTENT-list.lst
+++ b/CONTENT-list.lst
@@ -1,2 +1 @@
-CONTENT/SUITE/TopMenu
-en linje mere
+CONTENT/SUITE/TopMenu
\ No newline at end of file
-- 
1.8.0.msysgit.0

