From d2bbdd44c80bd0db1cbfcfeb130781594a48692c Mon Sep 17 00:00:00 2001
From: HansHenrikLund <hhl@dbc.dk>
Date: Fri, 4 Jan 2013 16:08:07 +0100
Subject: [PATCH 2/2] Nu virker det

---
 CONTENT-list.lst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CONTENT-list.lst b/CONTENT-list.lst
index 1f4ba9c..a332ad5 100644
--- a/CONTENT-list.lst
+++ b/CONTENT-list.lst
@@ -1 +1 @@
-CONTENT/SUITE/TopMenu
\ No newline at end of file
+CONTENT/SUITE/TopMenu
-- 
1.8.0.msysgit.0

